teachertool/src/App.tsx

```diff
@@ -30,13 +32,40 @@ function App() {
         }
     }, [ready]);
 
+    const onIframeLoaded = () => {
+        console.log("iframe loaded");
+    }
+
+    const onIframeClosed = () => {
+        console.log("iframe closed");
+    }
+
+    const createIFrameUrl = (): string => {
+        const editorUrl: string = isLocal() ? "http://localhost:3232/index.html#editor" : getEditorUrl((window as any).pxtTargetBundle.appTheme.embedUrl);
+
+        let url = editorUrl
+        if (editorUrl.charAt(editorUrl.length - 1) === "/" && !isLocal()) {
+            url = editorUrl.substr(0, editorUrl.length - 1);
+        }
+        url += `?controller=1&ws=browser&nocookiebanner=1`;
+        return url;
+    }
+
     return (
         <>
             <HeaderBar />
-            <div className="appContainer">
-                <DebugInput />
+            <div className="app-container">
+                <div className="debug-container">
```

Comment:
Can we move the debug-container div inside the DebugInput component itself? Just thinking it might be good to keep it all contained there.